src/C-DEngine/C-DEngines/C-DThingService/cdeTheThingRegistry.cs

```diff
@@ -479,6 +479,18 @@ public  static bool UnregisterThingGlobally(TheThing pThing)
             return true;
         }
 
+
+        static DateTimeOffset _lastGlobalThingRequest = DateTimeOffset.MinValue;
+        public static bool RequestGlobalThings()
+        {
```

Comment:
Shall we keep this internal for now?

Reply:
Plug-ins may want to call this on startup (like our config plug-in). Better than sending the TSM directly, so we can throttle?